src/BrightIdRegister.ts

```diff
@@ -0,0 +1,43 @@
+import { Address, BigInt, ethereum } from '@graphprotocol/graph-ts'
+import { Juror } from '../types/schema'
+import { BrightIdRegister as BrightIdRegisterContract } from '../types/AragonCourt/BrightIdRegister'
+import {
+  Register,
+} from '../types/templates/BrightIdRegister/BrightIdRegister'
+
+export function handleUserRegistered(event: Register): void {
+  updateJuror(event.params.sender, event)
+}
+
+function loadOrCreateJuror(jurorAddress: Address, event: ethereum.Event): Juror | null {
```

Comment:
i think that we are going to call juros as keepers

Reply:
I think we are going to call them participants. But i would prefer to wait until we reach a final decision, change terminology on the dashboard, and then change it here, i don't think it's high priority to change all terminology here as we would need to also change the rest of the main concepts.